src/common/selectors/deployComponentsSelector.ts

```diff
@@ -16,7 +16,8 @@ export async function selectDeployComponentsByAsyncOpId(
 ): Promise<DeployComponent[]> {
   const queryStr = `SELECT sf_devops__Source_Component__c, sf_devops__Operation__c, sf_devops__File_Path__c
                     FROM sf_devops__Deploy_Component__c  
-                    WHERE sf_devops__Deployment_Result__r.sf_devops__Status__c = '${asyncOpId}'`;
+                    WHERE sf_devops__Deployment_Result__r.sf_devops__Status__c = '${asyncOpId}'
+                    OR sf_devops__Deployment_Result__r.sf_devops__Check_Deploy_Status__c = '${asyncOpId}'`;
```

Comment:
It's early, and I have not had all my coffee yet so I cannot think of a concrete case, but this seems error prone.  Is there a case where we would get the wrong set of Deploy_Components because of the OR?

Do we just want to create a second selector to be more specific?  selectDeployComponentsForCheckDeployByAsynchOpId?

Reply:
Yes !!